gtda/mapper/utils/visualization.py

```diff
@@ -138,8 +138,11 @@ def _get_node_colors(data, is_data_dataframe, node_elements,
             node_elements, color_data, summary_stat=node_color_statistic)
 
     # Normalise node colours in range [0,1] for colorscale mapping
-    node_colors = (node_colors - np.min(node_colors)) / \
-        (np.max(node_colors) - np.min(node_colors))
+    if any(np.logical_not(np.isfinite(node_colors))):
+        from warnings import warn
+        warn('The variable defining colors contains NaN values.', RuntimeWarning)
```

Comment:
What do you think about a warning like "NaN values detected in the array of Mapper node colors! These values will be ignored in the color scale."

Reply:
Thanks, goodsuggestion. I believe I address all of your comments in [commit](https://github.com/giotto-ai/giotto-tda/pull/286/commits/dfb27df44a026e11dc5c5479001e8f1defff1819).